moler/cmd/unix/genericunix.py

```diff
@@ -15,12 +15,17 @@
 from moler.exceptions import CommandFailure
 from moler.helpers import remove_all_known_special_chars
 
-cmd_failure_causes = r'command not found|No such file or directory|running it may require superuser privileges|Cannot find device'
+cmd_failure_causes = ['command not found',
+                      'not found',
```

Comment:
'command not found' will be also cached by 'not found' so 'command not found' can be deleted.

Reply:
Isn't 'not found" too broad?

Reply:
Isn't 'not found" too broad?

Reply:
It appeared during testing adb shell while trying to switch into root via su command. On non-rooted android it appeared that there was no su command at all. Output was:
`su: not found`.
Question is: do we want it to be generic "error indication" or we will fix such cases one-by-one when specific command will timeout and logs show it should have been failed.

For me better approach is to let "not found" be error indication because:
1) if not, people may live with timeouts as "failed test case reason" without coming back, analysing logs and fixing/hardening commands.
2) if we set it and it should not be error indication since it is normal part of some output then test using it will fail and that will force people to fix it ;-)